lbz/events/broker.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import Callable, Dict, List
+
+from lbz.misc import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass()
+class Event:
+    type: str
+    data: dict
+
+
+class EventBroker:
+    def __init__(self, mapper: Dict[str, List[Callable]], raw_event: dict) -> None:
+        self.mapper = mapper
+        self.event = Event(type=raw_event["detail-type"], data=raw_event["detail"])
+
+    def handle(
+        self,
+    ) -> None:
+        self.pre_handle()
+
+        for handler in self._get_handlers():
+            try:
+                handler(self.event)
+            except Exception as error:  # noqa pylint: disable=broad-except, unused-variable
```

Comment:
Don't create an `error` variable if you don't use it 😉 

Thanks to that, you will be able to remove skipping `unused-variable`.